prime-router/metadata/test_fhir_mapping/hl7/message/ORU_R01.yml

```diff
@@ -0,0 +1,12 @@
+resources:
+
+  - resourceName: MessageHeader
+    segment: MSH
+    resourcePath: segments/MSH/MessageHeader
+    repeats: false
+    isReferenced: true
+    additionalSegments:
+      - SFT
+
+
+# This test should only process MSH to produce minimal results
```

Comment:
This only differs from the normal mappings in that MSH is the only segment to be processed, which results in a much smaller output file.